e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/PipelineRunFormAO.java

```diff
@@ -299,6 +305,34 @@ private RunParameterAO clickAddParameter(String parameterType) {
         return new RunParameterAO(this, parameterIndex);
     }
 
+    public PipelineRunFormAO clickAddSystemParameter() {
+        return click(ADD_SYSTEM_PARAMETER)
+                .ensure(SYSTEM_PARAMETERS_DIALOG, visible);
+    }
+
+    private PipelineRunFormAO selectSystemParameter(String parameter) {
+        SelenideElement inputField = $(byXpath("//*[@placeholder='Parameter']"));
+        Utils.clearTextField(inputField);
+        Utils.clickAndSendKeysWithSlashes(inputField, parameter);
+        $(byText(parameter)).shouldBe(visible).click();
+        return this;
+    }
+
+    public PipelineRunFormAO selectSystemParameters(String ... parameters) {
+        Arrays.stream(parameters).forEach(this::selectSystemParameter);
+        $(byId("system-parameters-browser-ok-button")).shouldBe(visible).click();
+        return this;
+    }
+
+    public PipelineRunFormAO inputSystemParameterValue(String parameter, String value) {
+        String inputFieldID = $(byXpath(String.format("//input[@value='%s']", parameter))).attr("id")
+                .replace(".name", ".value");
+        SelenideElement inputField = $(byXpath(String.format("//input[@id='%s']", inputFieldID)));
+        inputField.shouldBe(enabled);
+        Utils.sendKeysByChars(inputField, value);
+        return this;
+    }
+
```

Comment:
It would be better to create a new class for system parameter. For example, SystemParameterPopupAO, that extends PopupAO class

Reply:
Done